pootle/core/search/broker.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from . import SearchBackend
+
+import importlib
+
+from django.conf import settings
+
+
+class SearchBroker(SearchBackend):
+    def __init__(self, config_name=[]):
+        self._setup_settings()
```

Comment:
I see all subclasses of `SearchBackend` are calling its `_setup_settings()` method when initting. Why not just do it in `SearchBackend`'s `__init__()` and let subclasses call `super()`? It's a *private* method of the super class after all, children classes shouldn't be dealing with it.